dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php

```diff
@@ -210,6 +210,26 @@ public function testCodeStyle()
         );
     }
 
+    /**
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+     */
+    public function testCodeStyleWithPhpCsFixer()
+    {
+        $fixCommand = './vendor/bin/php-cs-fixer fix . --config=.php_cs.dist';
+        $checkCommand = $fixCommand . ' --dry-run 2> /dev/null';
+
+        exec($checkCommand, $output, $exitCode);
```

Comment:
Suppressing warnings for unused local variables here because I need to pass in `$output` (never used) to be able to assert on `$exitCode`.